Makefile

```diff
@@ -108,6 +108,16 @@ flowauth-build:
 	docker-compose -f $(DOCKER_COMPOSE_FILE) -f $(DOCKER_COMPOSE_FILE_BUILD) build flowauth
 
 
+worked-examples-up: worked-examples-build
```

Comment:
`worked_examples-up` instead of `worked-examples-up` would be more consistent with the names of the other targets.